components/compliance-service/reporting/constants.go

```diff
@@ -3,16 +3,16 @@ package reporting
 const ESize = 999999
 
 const (
-	UNREPORTED = "unreported"
+	UNREPORTED  = "unreported"
 	UNREACHABLE = "unreachable"
 	UNCOLLECTED = "uncollected"
-	COLLECTED = "collected"
+	COLLECTED   = "collected"
 )
 
 // These are filter types where we use ElasticSearch Term Queries
 var FilterType = []string{"environment", "organization", "chef_server", "chef_tags",
 	"policy_group", "policy_name", "status", "node_name", "platform", "platform_with_version",
-	"role", "recipe", "inspec_version", "ipaddress", "profile_id", "resource_type"}
```

Comment:
This is the filter the backend gets when we apply the filter called `Profile` on UI, why are we removing this?

Reply:
Hey @Durga, this is the filters type objects where we dont have nested conditions. Profile_id is removed now, because we are handling it in the filters method.

Reply:
these are all the node level filters. Other than the nested ones.